lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -152,10 +152,14 @@ class CourseCardModel extends Backbone.Model {
   formatDateString(run) {
     const pacingType = run.pacing_type;
     let dateString;
-    const start = CourseCardModel.valueIsDefined(run.start_date) ?
-      run.advertised_start || run.start_date :
-      this.get('start_date');
-    const end = CourseCardModel.valueIsDefined(run.end_date) ? run.end_date : this.get('end_date');
+    const start = CourseCardModel.valueIsDefined(run.start) ?
```

Comment:
Shouldn't we check for `start_date` and `end_date` first then fallback to start and end to meet the current implementation expectations?

Reply:
I am made the change. Kindly review.